code/src/components/MoviePage.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+
+import { MOVIE_DETAILS_URL } from 'reusable/Urls'
+import { ButtonBack } from './ButtonBack'
+
+export const MoviePage = () => {
+ 
+  const [movie, setMovie] = useState([])
+  const { id } = useParams()
+
+ useEffect(() => {
+   fetch( MOVIE_DETAILS_URL (id))
+   .then ((res) => res.json())
+   .then((json) => setMovie(json))
+   .catch((error) => console.log('Request failed', error))
```

Comment:
console.log should be removed before production, perhaps the code coach could give some suggestions on what to replace this with.